### PR TITLE
Added support for Answers and Definitions in ddg command

### PR DESCRIPTION
**Pull request by IceDragon200**
Formatting has changed slightly as well.

### CODE REVIEW

File: commands/ddg.rb [hunk 1]
```diff
@@ -1,25 +1,67 @@
...
-      abstract = data['Abstract'].presence
-      redirect = data['Redirect'].presence
-      u = url || redirect
-      if u && heading
-        b.reply "#{heading} #{b.fmt.uri(u)}"
-      elsif u
-        b.reply b.fmt.uri(u)
-      elsif heading
-        b.reply heading
+      func = if data['Abstract'].present?
```
Comment (archSeer) [inline commands/ddg.rb]: Couldn't this be a `case` statement? And you can skip the `.present?`

Comment (IceDragon200) [inline commands/ddg.rb]: The json request may return empty strings as the values instead of `nil`

Comment (archSeer) [inline commands/ddg.rb]: The case statement comment still stands

Comment (IceDragon200) [inline commands/ddg.rb]: @archSeer: An example of such a case statement if you please good sir

Comment (archSeer) [inline commands/ddg.rb]:
````
case
when data['Abstract'].present?
when data['Answer'].present?
when data['Definition'].present?
else
end
```

Hmm, that might not actually make it better.

Comment (IceDragon200) [inline commands/ddg.rb]: One day we'll figure it out

**Pull request merged**